api/client.yaml

```diff
@@ -2040,6 +2032,234 @@ components:
         - ReceiveOnly
         - Verified
         - Frozen
+    SICCode:
```

Comment:
Here and naics, should we use ints?
```
   type: integer
   format: int32
```

Reply:
Great question @vxio.  Since a few of them have a `0` prepended, such as `07`, we need to use string